src/app/components/terms-and-conditions/terms-and-conditions.component.spec.ts

```diff
@@ -31,20 +34,25 @@ describe('TermsAndConditionsComponent', () => {
     let component: TermsAndConditionsComponent;
     let fixture: ComponentFixture<TermsAndConditionsComponent>;
     let element: DebugElement;
+    let termsConditionsService: TermsConditionsService;
 
     beforeEach(async(() => {
         pipeSpy = spyOn(storeMock, 'pipe');
         dispatchSpy = spyOn(storeMock, 'dispatch');
         TestBed.configureTestingModule({
             imports: [
-                RouterTestingModule
+                RouterTestingModule,
+                HttpClientModule
```

Comment:
Why is this not using the testing module for http?

Reply:
Because I have injected termsConditionsService in the component, and termsConditionsService use HttpClient. so it throws error if I didn't import the HttpClientModule

Reply:
Change it to HttpClientTestingModule